2-structured-data/package.json

```diff
@@ -40,7 +40,6 @@
   "dependencies": {
     "body-parser": "^1.15.2",
     "express": "^4.14.0",
-    "gcloud": "~0.37.0",
     "@google-cloud/datastore": "^0.1.1",
```

Comment:
Per semver these will not necessarily be backwards compatible. Pre 1.0.0, semver allows for breaking changes in the minor version number.